apps/trading-e2e/src/integration/capsule.cy.ts

```diff
@@ -116,26 +134,144 @@ describe('capsule', { tags: '@slow' }, () => {
         });
         checkIfDataAndTimeOfCreationAndUpdateIsEqual(orderUpdatedAt);
       });
-
+    //cancel order
+    cy.getByTestId(ordersTab).click();
     cy.getByTestId('cancel').first().click();
-
-    cy.getByTestId('dialog-title').should(
+    cy.getByTestId(toastContent).should(
       'contain.text',
-      'Awaiting network confirmation'
+      `ConfirmedYour transaction has been confirmed View in block explorerCancel order - cancelledTEST.24h+0.0005 @ 200.00 ${usdcSymbol}`
     );
-    cy.getByTestId('dialog-title').should('contain.text', 'Order cancelled');
-    cy.getByTestId('dialog-close').click();
+    cy.getByTestId(toastCloseBtn).click();
 
     cy.getByTestId('tab-orders')
       .get('.ag-center-cols-container')
       .children()
       .first()
-      .get(`[col-id='${orderStatus}']`)
+      .get(`[col-id='${orderStatus}']`, txTimeout)
       .should('contain.text', OrderStatusMapping.STATUS_CANCELLED);
   });
+
+  it('can deposit and withdrawal', function () {
+    // 1001-DEPO-001
+    // 1001-DEPO-002
+    // 1001-DEPO-003
+    // 1001-DEPO-005
+    // 1001-DEPO-006
+    // 1001-DEPO-007
+    // 1001-DEPO-008
+    // 1001-DEPO-009
+    // 1002-WITH-001
+    // 1002-WITH-006
+    // 1002-WITH-009
+    // 002-WITH-011
+    // 1002-WITH-024
+    // 1002-WITH-012
+    // 1002-WITH-013
+    // 1002-WITH-014
+    // 1002-WITH-015
+    // 1002-WITH-016
+    // 1002-WITH-019
+
+    cy.visit('/#/portfolio');
+    cy.get('main[data-testid="/portfolio"]').should('exist');
+
+    cy.highlight('creating deposit');
+
+    cy.getByTestId(depositsTab).click();
+    cy.getByTestId('deposit-button').click();
+    connectEthereumWallet('Unknown');
+    cy.get(assetSelectField, txTimeout).select(btcName);
+    cy.getByTestId('deposit-approve-submit').click();
+    cy.getByTestId('dialog-title').should('contain.text', 'Approve complete');
+    cy.get('[data-testid="Return to deposit"]').click();
+    cy.get(amountField).clear().type('1');
+    cy.getByTestId('deposit-submit').click();
+    cy.getByTestId(toastContent, txTimeout).should(
+      'contain.text',
+      `Transaction completedYour transaction has been completedView on EtherscanDeposit 1.00 ${btcSymbol}`
```

Comment:
suggestion (non blocking): This is a little icky. Presumably the text is broken up by different elements `contain.text` treats it all as a joined string. I think this would be nicer if we targeted a single element containing the toast state which we can then assert is 'complete'

Reply:
Yes we can do it, but this is to check if everything really works e2e. This method found us a bug #2695